### PR TITLE
Create data migration for first step of schema transition

### PR DESCRIPTION
**Pull request by awenocur**
Note that this depends on [PR 2](https://github.com/cbmi/varify-data-warehouse/pull/2).  The attached PR shall be invalid until this merge is complete.

This shall be a reversible data migration to complete the first step of the greater transition to a new schema.

### CODE REVIEW

File: vdw/samples/migrations/0036_split_variants.py [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        all_results = orm.Result.objects.all()
+        for result in all_results:
+            if result.genotype.label == '#/#' or\
```
Comment (naegelyd) [inline vdw/samples/migrations/0036_split_variants.py:12]: Consider changing this to the following:

``` python
if result.genotype.label in ['#/#', '1/1', '1/2', '2/2#']:
    ...
```

File: vdw/samples/migrations/0036_split_variants.py [hunk 2]
```diff
@@ -0,0 +1,316 @@
+# -*- coding: utf-8 -*-
+import datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        all_results = orm.Result.objects.all()
```
Comment (naegelyd) [inline vdw/samples/migrations/0036_split_variants.py:10]: Blank line below

File: vdw/samples/migrations/0036_split_variants.py [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+                result.genotype.label == '1/1' or\
+                result.genotype.label == '1/2' or\
+                result.genotype.label == '2/2#':
+                    # Reference is absent; use only the second "allele". It may
+                    # be a comma-delimited list of ref alleles at this point.
+                    variant_to_dissect = result.allele_2
+                    result.allele_2 = None
+                    alleles = variant_to_dissect.alt.split(',')
+            else:
+                    result.allele_2 = None
```
Comment (naegelyd) [inline vdw/samples/migrations/0036_split_variants.py:22]: What is the purpose of this line? 3 lines down, you set this same this to `None`. I'm guessing this is a duplicate, remove it and the blank line below it.

**Comment by awenocur** [comment]
This migration should be bundled with the following:
- update vdw.samples.models.compute_allele_frequencies to take into account het alt results:
  - The INSERT statement that populates the cohort_variant table should be duplicated to find results that reference an allele from either foreign key.
  - This is about as messy as the transition should become, prior to the foreign key pair being migrated to a lookup table.
- update vdw.samples.pipeline.utils.process_line to generate link entries in the Variant table for both alleles, adding `allele_2_id` to `output_columns`

**Pull request closed without merging**